lndr-backend/src/Lndr/Db/Types.hs

```diff
@@ -22,6 +22,17 @@ instance FromField Address where
 
 instance FromRow UserInfo
 
+instance FromRow PayPalRequestPair where
+    fromRow = do
+        targetAddr <- field
+        requestorAddr <- field
+        targetNick <- field
+        requestorNick <- field
```

Comment:
:confused: Where does `field` come from / where is it defined?

Reply:
`field` comes from the PostgreSQL library we're using and it  maps to the order of the columns in the SELECT statement that returns PayPalRequestPair